stl/inc/yvals_core.h

```diff
@@ -109,6 +109,8 @@
 // P0682R1 Repairing Elementary String Conversions
 // P0739R0 Improving Class Template Argument Deduction For The STL
 // P0858R0 Constexpr Iterator Requirements
+// P1065R2 constexpr INVOKE
+//   (the std::invoke function only; other components like bind and reference_wrapper will be C++20 only)

```

Comment:
This is unusual because (1) it's currently partial and (2) you're planning to split it across 17 and 20. I guess if it's later repeated under `_HAS_CXX20` with a complementary comment, that's OK.

Reply:
Yes, a second entry under _HAS_CXX20 is the intent.